src/assets/JSAsset.js

```diff
@@ -26,6 +25,17 @@ class JSAsset extends Asset {
     this.isAstDirty = false;
     this.isES6Module = false;
     this.outputCode = null;
+    this.cacheData.env = {};
+  }
+
+  shouldInvalidate(cacheData) {
```

Comment:
This causes an error on existing builds (as they already have cache files) and cacheData is undefined in these files.

Reply:
hmm yeah, but should be ok with the npm version since the parcel version is part of the cache key right? so when people update parcel, everything will be invalidated already.

Reply:
Ow forgot about that, yeah should be fine